curriculum/challenges/english/25-front-end-development/quiz-relational-database/66f1af82732957c895f0b21a.md

```diff
@@ -17,439 +17,440 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a relational database?
 
 #### --distractors--
 
-Placeholder distractor 1
+A database structured to recognize non-relation between stored items
 
 ---
 
-Placeholder distractor 2
+A database that stores unstructured data
 
 ---
 
-Placeholder distractor 3
+A flat file-based system
 
 #### --answer--
 
-Placeholder answer
+A database structured to recognize relations between stored items
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How does a non-relational database differ from a relational database?
 
 #### --distractors--
 
-Placeholder distractor 1
+Non-relational databases use tables, while relational databases do not
 
 ---
 
-Placeholder distractor 2
+Relational databases can store unstructured data
 
 ---
 
-Placeholder distractor 3
+Non-relational databases are slower than relational databases
 
 #### --answer--
 
-Placeholder answer
+Non-relational databases can store unstructured data
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the PostgreSQL command \l do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Lists all schemas
 
 ---
 
-Placeholder distractor 2
+Lists all tables
 
 ---
 
-Placeholder distractor 3
+Connects to a database
 
 #### --answer--
 
-Placeholder answer
+Lists all databases
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following commands creates a new table in SQL?
 
 #### --distractors--
 
-Placeholder distractor 1
+ADD TABLE
 
 ---
 
-Placeholder distractor 2
+NEW TABLE
 
 ---
 
-Placeholder distractor 3
+ALTER TABLE
 
 #### --answer--
 
-Placeholder answer
+CREATE TABLE
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of a foreign key in a relational database?
 
 #### --distractors--
 
-Placeholder distractor 1
+To allow duplication of data
 
 ---
 
-Placeholder distractor 2
+To uniquely identify a row in the table
 
 ---
 
-Placeholder distractor 3
+To store large amounts of text data
 
 #### --answer--
 
-Placeholder answer
+To create a relationship between two tables
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command will add a new column to an existing table?
 
 #### --distractors--
 
-Placeholder distractor 1
+MODIFY TABLE
 
 ---
 
-Placeholder distractor 2
+INSERT COLUMN
 
 ---
 
-Placeholder distractor 3
+ADD COLUMN
 
 #### --answer--
 
-Placeholder answer
+ALTER TABLE
```

Comment:
Is this complete command which would add new column, or just part of it?

Reply:
"ALTER TABLE" is only part of the command.To add a new column to an existing table in PostgreSQL

Reply:
I think it might be worth considering writing whole example command - `ALTER TABLE table_name ADD COLUMN column_name;`. Similarly to the other question.